### PR TITLE
CLI improvements

### PR DESCRIPTION
**Pull request by aharpervc**
Here's some improvements I came up with while working on other things. Per the changelog:

* Stop with error when force rendering and no template is found
* Remove non-functional short verbosity argument
* Add `--version` option to show the version
* Better logging for template rendering status, errors, and config (verbose mode)
* Development: adjust require's to be able to run CLI from any folder

### CODE REVIEW

File: lib/consult/template.rb [hunk 1]
```diff
@@ -21,12 +21,13 @@ def render(save: true)
       renderer = ERB.new(contents, nil, '-')
       result = renderer.result(binding)
 
+      puts "Consult: rendering #{name}..." if verbose?
```
Comment (henrythach) [inline lib/consult/template.rb]: To be consistent with the other logging messages, I think we should capitalize the first word after `Consult: `, e.g.

```suggestion
      puts "Consult: Rendering #{name}..." if verbose?
```

Comment (henrythach) [reply lib/consult/template.rb]: While we're here in the name of CLI improvements, would it be useful to include the path of where it is rendered?

For example:

```suggestion
      puts "Consult: rendering #{name} to #{path}..." if verbose?
```

Example output:

```
$ bundle exec consult
Consult: Loading config from /Users/henry.thach/project/config/consult.yml
Consult: rendering foo to /Users/henry.thach/project/config/some/path/to/foo.yml...
Consult: rendering bar to /Users/henry.thach/project/config/bar.yml...
Consult: rendering baz to /Users/henry.thach/project/whatever/baz.yml...
```

**Review by henrythach — COMMENTED** [review]
Overall looks good to me.  I was able to see the new error messages, as well as silence them with `--quiet`.

I just left two comments regarding consistency with the logging format and a recommendation to add the path to the output file.

File: lib/consult/template.rb (hunk 1 above)
Comment (aharpervc) [reply lib/consult/template.rb]: Ha, that's exactly how I had it in an earlier iteration. I reverted it because it was a bit too verbose but I'll restore it now with your vote

Comment (ronnietaylor) [reply lib/consult/template.rb]: Just had a chance to test. I agree that showing the path of the rendered file is helpful.

File: lib/consult.rb [hunk 2]
```diff
@@ -6,9 +6,9 @@
 require 'vault'
 require 'diplomat'
 
-require 'consult/version'
-require 'consult/utilities'
-require 'consult/template'
+require_relative './consult/version'
+require_relative './consult/utilities'
+require_relative './consult/template'
```
Comment (ronnietaylor) [inline lib/consult.rb:11]: Why is this change needed?

Comment (aharpervc) [reply lib/consult.rb:11]: It's to enable running consult from any directory, from source. Eg, go to some app folder and run `ruby ..\consult\bin\consult`

**Review by henrythach — APPROVED** [review]
LGTM!

**Review by ronnietaylor — APPROVED** [review]
These changes seem sane. The changes related to `version` and `quiet` work as expected for me as well.
